daemon/vpn/openvpn/config.go

```diff
@@ -74,15 +74,16 @@ func generateConfigFile(protocol config.Protocol, serverIP netip.Addr, obfuscate
 
 	template, err := internal.FileRead(templatePath)
 	if err != nil {
-		return fmt.Errorf("reading ovpn template file")
+		return fmt.Errorf("reading ovpn template file: %w", err)
 	}
 
 	out, err := generateConfig(serverIP, identifier, template)
 	if err != nil {
 		return fmt.Errorf("generating OpenVPN config: %w", err)
 	}
 
-	if err := addExtraParameters(out, serverIP, protocol); err != nil {
+	out, err = addExtraParameters(out, serverIP, protocol)
```

Comment:
So the result of `addExtraParameters` was basically not used and now it is being used 🤔 , are there any differences in the configurtion?